src/node_kernels/BLTGammaM2015NodeKernel.C

```diff
@@ -120,11 +115,18 @@ BLTGammaM2015NodeKernel::execute(
   DblType sijMag = 0.0;
   DblType vortMag = 0.0;
 
-  DblType Ctu1 = 100.;
-  DblType Ctu2 = 1000.;
-  DblType Ctu3 = 1.0;
+  // constants for the source terms
```

Comment:
These are always constant and should be hardcoded? Just checking

Reply:
Discussed.